commands/list.go

```diff
@@ -57,8 +57,8 @@ func runList(cmd *cobra.Command, args []string) {
 	// fmt.Println(gatewayAddress)
 	functions, err := proxy.ListFunctions(gatewayAddress)
 	if err != nil {
-		log.Println(err)
-		return
+		// ListFunctions prints out its own error messages
+		return err
```

Comment:
I would be tempted to just return an empty error here as updating the proxy would still leave the FaaS-netes provider requiring an update. Returning `fmt.Errorf("")` minimises the impact.

@alexellis wdyt?